### DISCUSSION TITLE
server运行时出现这个错误

### DISCUSSION THREAD

**Discussion by monster0147** [AlistGo/alist#2013] [Q&A]
failed to start: listen tcp 0.0.0.0:5244: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

**Comment by jiake568** [comment]
failed to start: listen tcp 0.0.0.0:5244: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

**Comment by anwen-anyi** [comment]
很明显你都已经运行一个了，端口号冲突了 =，= 翻译一下就知道什么意思啦~

**Reply by KBKUN024** [reply]
并没有，查了端口号，5244没有被占用，今天莫名其妙出现这个问题
